br/pkg/task/common.go

```diff
@@ -209,6 +210,8 @@ type Config struct {
 
 	CipherInfo backuppb.CipherInfo `json:"-" toml:"-"`
 
+	// determines whether enable restore sys table on default, see fullClusterRestore in restore/client.go
+	WithSysTable bool `json:"with-sys-table" toml:"with-sys-table"`
```

Comment:
I think only restore need this config, so you can move it to `RestoreCommonConfig`